src/module/actor/pokemon/document.js

```diff
@@ -354,27 +354,55 @@ class PTUPokemonActor extends PTUActor {
 
     // TODO: Implement rules for capability changing items
     _calcCapabilities() {
-        const capabilities = duplicate(this.species?.system?.capabilities ?? {});
-        if (!capabilities) return {};
+        const numericNonMovementCapabilities = CONFIG.PTU.Capabilities.numericNonMovement
+        const stringArrayCapabilities = CONFIG.PTU.Capabilities.stringArray
+
+        const speciesCapabilities = duplicate(this.species?.system?.capabilities ?? {});
+        const finalCapabilities = {}
+        if (!speciesCapabilities) return {};
+
+        // Anything that is not a part of numericNonMovementCapabilities or stringArrayCapabilities is considered
+        // movement, without explicitly listing them hardcoded.
+        const capsFromSpeciesOrModifiers = [].concat(Object.keys(this.system.modifiers.capabilities), Object.keys(speciesCapabilities))
```

Comment:
You can use spread operator here `[...Object.keys(this.system.modifiers.capabilities), ...Object.keys(speciesCapabilities)]`